spotifycli/spotifycli.py

```diff
@@ -58,7 +58,10 @@ def main():
         perform_spotify_action("Next")
     elif args.prev:
         perform_spotify_action("Previous")
-
+    elif args.songuri:
+        perform_spotify_action("OpenUri", f"string:spotify:track:{args.songuri}")
+    elif args.listuri:
+        perform_spotify_action("OpenUri", f"string:spotify:playlist:{args.listuri}")
```

Comment:
If songuri or listuri are specified, they will come in storing the argument value, so pass this to the expanded `perform_spotify_action` function.